bundles/org.eclipse.swt.svg/META-INF/MANIFEST.MF

```diff
@@ -11,5 +11,5 @@ Fragment-Host: org.eclipse.swt
 Import-Package: com.github.weisj.jsvg;version="[2.0.0,3.0.0)",
  com.github.weisj.jsvg.parser;version="[2.0.0,3.0.0)",
  com.github.weisj.jsvg.view;version="[2.0.0,3.0.0)"
-Export-Package: org.eclipse.swt.svg
+Export-Package: org.eclipse.swt.svg;x-internal:=true
```

Comment:
I think the commit message is confusing... this does not *remove* the export it only marks it as *internal* ...

Reply:
The export is only needed if you import it somewhere of course... not sure where is is really used, if its only for test thex look to me that they can be plain surefire tests without OSGI and then package exports should not matter at all (you might want jar.extra classpath instead to make PDE IDE happy)

Reply:
The only usage is in `org.eclipse.swt.test` and there we don't use a package import, so it should not matter? At least I do not see any complain in my IDE when simply removing the export here.
And yes, those tests could be plain surefire tests and could also be placed inside that fragment, as discussed in the comments above: https://github.com/eclipse-platform/eclipse.platform.swt/pull/2602#discussion_r2411310740

Reply:
Okay, the build properly complains, only my IDE did not. I changed to `x-internal` again and adapted the commit message.